### PR TITLE
fix: WCAG color contrast issue in footer

### LINKED ISSUE #386: [BUG] color contrast issue fix (WCAG color contrast issue)
### Description

**Requirements:**

Our app should pass all the WCAG color contrast checks. I've added a sample screenshot for reference.

Please install WCAG [Color contrast checker](https://chrome.google.com/webstore/detail/wcag-color-contrast-check/plnahcmalebffmaghcpcmpaciebdhgdf/related?hl=en) extension and fix all the issues in every single page. 
It should pass all the checks in all pages.

### Screenshots

Screenshot 2023-05-04 at 10 55 52 PM

### PR DESCRIPTION
**Pull request by Lucif3r-in**
### Fixes Issue
Close #386 

### Note to reviewers
Changed to color of text from `text-gray-500` and `hover:text-gray-400` to  `text-gray-400` and `hover:text-gray-300`

Fixes the WCAG contrast issue

### CODE REVIEW

**Comment by Lucif3r-in** [comment]
@rohitdasu Can you please review this?

**Comment by rohitdasu** [comment]
hi @Lucif3r-in did you fix it in all pages?

**Comment by Lucif3r-in** [comment]
Ya I changed in the footer component so it would reflect on every page. @rohitdasu . Lemme check for other contrast issues.

**Comment by rohitdasu** [comment]
> Ya I changed in the footer component so it would reflect on every page. @rohitdasu . Lemme check for other contrast issues.

not just for footer. please check it in every page and component using the chrome extension.

**Comment by Lucif3r-in** [comment]
@rohitdasu Please review!! I have resolved all the contrast issues.

File: pages/mates/index.tsx [hunk 1]
```diff
@@ -1,15 +1,54 @@
...
+          {profiles.map((profile) => (
+            <div key={profile.id} className="w-1/2 p-4 sm:w-1/4 md:w-1/6">
+              <div className="rounded-lg bg-white p-6 shadow">
+                <Image
+                  src={profile.profilePicture}
+                  alt={profile.name}
+                  className="mx-auto mb-4 h-16 w-16 rounded-full"
+                />
+                <h2 className="text-lg font-bold">{profile.name}</h2>
+              </div>
```
Comment (rohitdasu) [inline pages/mates/index.tsx:48]: it's not related to this issue.

Comment (Lucif3r-in) [reply pages/mates/index.tsx]: Sorry about that, I was working on some other issue. I have made the required changes can you please review it? @rohitdasu

**Comment by Lucif3r-in** [comment]
Sorry about that, I was working on some other issue. I have made the required changes can you please review it? @rohitdasu

**Review by rohitdasu — APPROVED** [review]
LGTM!

**Pull request merged**